Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -162,6 +161,16 @@ for multiplication in a normed algebra over the base field. -/
 theorem differentiable_circleMap (c : ℂ) (R : ℝ) : Differentiable ℝ (circleMap c R) := fun θ =>
   (hasDerivAt_circleMap c R θ).differentiableAt
 
+open Complex in
```

Comment:
```suggestion
```
Not needed, I think.

Reply:
You are right. Delted line as suggested.